### ISSUE TITLE
Profile picture resists conforming into a circular shape

### ISSUE THREAD

**Issue by paniash** [paniash/website#6]
Currently the profile picture is placed to the right with its original aspect ratio. The goal is to make it circular in a fashion similar to [Rik Huijzer's website](https://huijzer.xyz/).

The same code for adding the image (html + css) is being used to circularize the current profile picture but it turns out to be in an oval fashion (looks more like a guitar pick). I tried testing the exact same code with Rik's profile picture and the same issue persists, so it's definitely not an issue with the aspect ratio of the image used.

Not sure what the issue is when I'm exactly copying Rik's [code](https://gitlab.com/rikh/blog) specifically in `franklin.css`:

```css
.avatar-image {
  width: 8rem !important;
  height: auto;
  border-radius: 50%;
}
```

and in `index.md`

```html
~~~
<center>
<img class="avatar-image" src="/assets/images/self.jpg">
</center>
~~~
```

Could you help me out? @tlienart

## Screenshot

Using Rik's profile picture, the present issue:

![pic-selected-220101-2323-03](https://user-images.githubusercontent.com/59497618/147857014-a1461da1-47d0-40d8-94d2-98e136bda008.png)

**Comment by tlienart** [comment]
@paniash sorry but these are all pretty basic HTML and CSS issues and unrelated to Franklin, I think you'd be better off using Stackoverflow to get answers; have a look at https://www.webfx.com/blog/web-design/circular-images-css/

Generally with such stuff you should use the developer tools from your browser and see whether some css rules are invalidated; if they are then you should try to remove them and see whether it helps and iterate.

**Comment by paniash** [comment]
@tlienart Thanks! There was some issue with the padding setting under `.franklin-content img`. Setting it to 0 fixed the issue. :-)

**Comment by paniash** [comment]
Fixed in 1c068d7a5d3289b70e70e180f4dd96b5e76a27e5.
